deployment/katanaDeployment.js

```diff
@@ -0,0 +1,135 @@
+require('@nomiclabs/hardhat-ethers');
+const fs = require('fs');
+const path = require('path');
+const configPath = path.join(__dirname, './katana_input.json');
+const configParams = JSON.parse(fs.readFileSync(configPath, 'utf8'));
+
+const keypress = async () => {
+  process.stdin.setRawMode(true);
+  return new Promise((resolve) =>
+    process.stdin.once('data', (data) => {
+      const byteArray = [...data];
+      if (byteArray.length > 0 && byteArray[0] === 3) {
+        console.log('^C');
+        process.exit(1);
+      }
+      process.stdin.setRawMode(false);
+      resolve();
+    })
+  );
+};
+
+async function pressToContinue() {
+    console.log("Checkpoint... Press any key to continue!");
+    await keypress();
+  }
+
+let kncAddress;
+let deployerAddress;
+let epochPeriod;
+let starttime;
+let shortExecutorConfig;
+let longExecutorConfig;
+let daoOperator;
+let outputFilename;
+
+task('deployGovInfra', 'deploys staking, governance, voting power strategy and executors').setAction(async () => {
+  parseInput(configParams);
+  const [deployer] = await ethers.getSigners();
+  deployerAddress = await deployer.getAddress();
+
+  // contract deployment
+  console.log('deploying staking contract...');
+  const KyberStaking = await ethers.getContractFactory('KyberStaking');
```

Comment:
we need to staking.updateWithdrawHandler to voitingPowerStrategy

Reply:
ok thanks